exercises/forth/forth_test.go

```diff
@@ -0,0 +1,53 @@
+package forth
+
+// API:
+// func Forth([]string) ([]int, error)
+//
+
+import "testing"
+
+const targetTestVersion = 1
+
+type testCase struct {
+	description string
+	input       []string
+	expected    []int // nil slice indicates error expected.
+}
+
+func sliceEqual(a, b []int) bool {
```

Comment:
Any strong preferences on using this `sliceEqual` vs https://golang.org/pkg/reflect/#DeepEqual ? Obviously we can tell that `sliceEqual` does exactly what we want and now can avoid having to import `reflect`, but interested to hear.